GitFyle.Core.Api/Program.cs

```diff
@@ -31,6 +32,9 @@ public static void Main(string[] args)
                 ILoggingBroker,
                 LoggingBroker>();
 
+                IDateTimeBroker, 
```

Comment:
Register the DateTimeBroker with the AddTransient extenstion method, will be the same as the registration done with the Storage and Logging Brokers.

Reply:
This has been updated. Please review to confirm issue has been resolved.

Reply:
Good catch on noticing this wasn't handled.